.github/workflows/pr-check.yml

```diff
@@ -0,0 +1,32 @@
+name: Test greentea-client
+
+on: [push, pull_request]
+
+jobs:
+  astyle:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install astyle
+      run: sudo apt-get install astyle
```

Comment:
Is `-y` needed or not for non-interactive installation?

Reply:
Ah, good point. It's not causing an issue currently, but yes we should use `-y`.